.github/workflows/test.yml

```
@@ -55,9 +55,25 @@ jobs:
         run: |
           pytest --doctest-modules --durations=10 --pyargs networkx
 
+  dispatch:
+    runs-on: ubuntu-latest
+    matrix:
+      python-version: ["3.11"]
```

Not sure why this isn't running any of the Github actions....  Is it because `strategy` is missing?  Why does that stop all the other actions?
```suggestion
    strategy:
      matrix:
        python-version: ["3.11"]
```

---

Yeah this fixed it. Thanks!